changelog/std-traits-issomestring.dd

```diff
@@ -0,0 +1,88 @@
+$(REF `isSomeString`, std, traits) and $(REF `isNarrowString`, std, traits) are now `false` for enums
+
+Previously, enums whose base type was a string type were `true` for
+$(REF `isSomeString`, std, traits) and $(REF `isNarrowString`, std, traits).
```

Comment:
Don't use backticks in `$(REF ...)`. REF already formats as code, and the backticks mess with the URL.